### ISSUE TITLE
Urdu Artificial Text Detection: Train and evaluate an image segmentation model (U-NETs) using from PyTorch or tensorflow or using the YOLO family of models.

### ISSUE THREAD

**Issue by akash-kamalesh** [dscpesu/SimpleML#11]
---

| Field  | Description                       |
| ------ | --------------------------------- |
| About  | A short Description about project |
| Github | Your Github name                  |
| Email  |                                   |
| Label  | Update request                    |

---

**Define You**

- [ ] Hacktober Fest Contributor

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like...**

**Describe alternatives you've considered?**

**Approach to be followed (optional):**

**Additional context**

**Comment by OmkarKashyap** [comment]
> Field	Description
> About	Train and evaluate an image segmentation model or any YOLO model to efficiently locate and delineate Urdu text in 
                 images.
> Github	OmkarKashyap
> Email	omkar.m.kashyap@gmail.com
> Label	Update request
> **Define You**
> 
> * [ ]  Hacktober Fest Contributor
> 
> **Is your feature request related to a problem? Please describe.**
> 
> **Describe the solution you'd like...**
>  Would need to look at data collection and preparation and implement various models and check for the Jaccard 
       index/Dice Coefficient and many other evaluation metrics to find the best model
> **Describe alternatives you've considered?**
> 
> **Approach to be followed (optional):**
>  Try and test out different YOO models along with trying U-NET 
> **Additional context**
I would like to work on this task , request to assign it to me.
Thanks
